src/Fields/Fields.jl

```diff
@@ -311,13 +311,38 @@ which reduces to
 ```
 """
 function Spaces.weighted_dss!(
-    field::Field,
+    field::Field{V, S},
     ghost_buffer = Spaces.create_ghost_buffer(field),
-)
+) where {V, S}
     Spaces.weighted_dss!(field_values(field), axes(field), ghost_buffer)
     return field
 end
 
+function Spaces.weighted_dss_start!(
+    field::Field,
+    ghost_buffer = Spaces.create_ghost_buffer(field),
```

Comment:
I don't think we should provide a default value for `ghost_buffer` for the split ones, as we need to make sure they are always the same for all three calls.